tests/time_tests/README.md

```diff
@@ -19,6 +18,14 @@ mkdir build && cd build
 cmake .. -DInferenceEngineDeveloperPackage_DIR=$(realpath ../../../build) && make time_tests
 ```
 
+If you don't have OpenVINO installed you need to have the `build` folder, which
+is created when you configure and build OpenVINO™.
+
+``` bash
+cmake .. -DInferenceEngineDeveloperPackage_DIR=$(realpath ../../../build) && make time_tests
```

Comment:
looks like duplication of command above: https://github.com/openvinotoolkit/openvino/pull/3484/files#diff-5e2b9b04ee95ba5077492b60d88e429b40c4992330b9d10be613f48797651ac4R18